images/cheatsheet/tmp.txt

```diff
@@ -0,0 +1 @@
+temp
```

Comment:
Is this file necessary?

Reply:
I needed that to uplaod the files :) deleted